data-raw/montlake-test-data.R

```diff
@@ -0,0 +1,145 @@
+## code to prepare `montlake_od_tbl`, `montlake_zone_tbl` and
+## `montlake_osm_buildings_tbl` dataset goes here
+library(tidyverse)
+library(sf)
+
+## determine the bounding box of the test area
+montlake_poly_url <- "https://raw.githubusercontent.com/a-b-street/abstreet/master/importer/config/us/seattle/montlake.poly"
```

Comment:
Usually I would protest the use of `<-` assignment in a package that uses 'equals style' but happy to let it pass this time as it's not actually shipped package code : )

Reply:
Although I may switch to `=` post merge if I'm feeling very pedantic :smile: